azure/functions/__init__.py

```diff
@@ -45,6 +45,10 @@
 
     # Middlewares
     'WsgiMiddleware',
+
+    # serialization
+    '__serialize_custom_object'
+    '__deserialize_custom_object'
```

Comment:
This is the export statement of `azure.functions`. I don't think we actually need these two lines here since the user of our library would not use `func.__serialize_custom_object` or `func.__deserialize_custom_object`.